filebeat/module/system/module.yml

```diff
@@ -0,0 +1,12 @@
+dashboards:
```

Comment:
@ruflin My idea was that we might want to include other configurations here as well in the future.  I agree to rename it to `dashboards.yml` and we can change the name to something else when we have other configurations to add to it.